### ISSUE TITLE
Fmovies fails to search or play anything

### ISSUE THREAD

**Issue by pitsi** [mbebe/blomqvist#824]
As mentioned in the title, fmovies fails to search or play anything and pops a urllib error in the log.
This is what the log says for the failed search
```
2022-08-23 19:15:01.512 T:140483493283584   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.IndexError'>
                                            Error Contents: list index out of range
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.video.fmoviesto/main.py", line 1205, in <module>
                                                router(sys.argv[2][1:])
                                              File "/storage/.kodi/addons/plugin.video.fmoviesto/main.py", line 1198, in router
                                                ListMovies(url,'1')
                                              File "/storage/.kodi/addons/plugin.video.fmoviesto/main.py", line 168, in ListMovies
                                                links, serials, pagin = getMovies(exlink,page)
                                              File "/storage/.kodi/addons/plugin.video.fmoviesto/main.py", line 223, in getMovies
                                                result = parseDOM(html, 'div', attrs={'class': "filmlist\s+.*?"}) [0]
                                            IndexError: list index out of range
                                            -->End of Python script error report<--
2022-08-23 19:15:01.686 T:140483347085056   ERROR: GetDirectory - Error getting plugin://plugin.video.fmoviesto/?image=DefaultAddonsSearch.png&mode=search&movie=True&moviescount=0&page=1&title=%5bCOLOR%20lightblue%5dSearch%5b%2fCOLOR%5d&url
2022-08-23 19:15:01.696 T:140483657521280   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.fmoviesto/?image=DefaultAddonsSearch.png&mode=search&movie=True&moviescount=0&page=1&title=%5bCOLOR%20lightblue%5dSearch%5b%2fCOLOR%5d&url) failed
```
And this is what it says for movies and series respectively, when selecting one from the main menu, right before showing the hosts selection dialog
```
2022-08-23 19:18:15.775 T:140482214594304   ERROR: GetDirectory - Error getting plugin://plugin.video.fmoviesto/?image=https%3a%2f%2fstatic.bunnycdn.ru%2fi%2fcache%2fimages%2f3%2f35%2f352e9aec44217a8b6394be18859821fb.jpg-w180&mode=getLinks&movie=True&moviescount=0&page=1&title=Run%20All%20Night&url=https%3a%2f%2ffmovies.to%2fmovie%2frun-all-night-dkw%7cdkw
2022-08-23 19:18:15.777 T:140483657521280   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.fmoviesto/?image=https%3a%2f%2fstatic.bunnycdn.ru%2fi%2fcache%2fimages%2f3%2f35%2f352e9aec44217a8b6394be18859821fb.jpg-w180&mode=getLinks&movie=True&moviescount=0&page=1&title=Run%20All%20Night&url=https%3a%2f%2ffmovies.to%2fmovie%2frun-all-night-dkw%7cdkw) failed

2022-08-23 19:20:27.772 T:140481979922176   ERROR: GetDirectory - Error getting plugin://plugin.video.fmoviesto/?image=https%3a%2f%2fstatic.bunnycdn.ru%2fi%2fcache%2fimages%2f2019%2f04%2f44e840d154ec960db6f7815009fa2fe7.jpg-w180&mode=getseasons&movie=True&moviescount=0&page=1&title=Game%20of%20Thrones%20Greatest%20Moments%20%5bCOLOR%20gold%5d(serie)%5b%2fCOLOR%5d&url=https%3a%2f%2ffmovies.to%2fseries%2fgame-of-thrones-greatest-moments-l3zn6%7cl3zn6
2022-08-23 19:20:27.794 T:140483657521280   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.fmoviesto/?image=https%3a%2f%2fstatic.bunnycdn.ru%2fi%2fcache%2fimages%2f2019%2f04%2f44e840d154ec960db6f7815009fa2fe7.jpg-w180&mode=getseasons&movie=True&moviescount=0&page=1&title=Game%20of%20Thrones%20Greatest%20Moments%20%5bCOLOR%20gold%5d(serie)%5b%2fCOLOR%5d&url=https%3a%2f%2ffmovies.to%2fseries%2fgame-of-thrones-greatest-moments-l3zn6%7cl3zn6) failed
```
The logs above are from kodi 18.9, but the same happens to kodi 19.4 as well. Both are libreelec x64 installations, 9.2.6 and 10.0.2 respectively.

**Comment by amerr97** [comment]
i have the same issue

**Comment by jferreira93-web** [comment]
Yeah, for months now, no one solves it.

**Comment by pitsi** [comment]
Well, if it has been like that for months, why haven't you or anyone else reported it so that the dev can check it?
I launch fmovies very rarely (actually when there is a new movie or series I really want to watch) so I reported the issue once I found it.

**Comment by pitsi** [comment]
Fixed with today's upgrade to v1.19.
Thank you very much @mbebe!

**Comment by ThomasCapella** [comment]
It shows subtitle selection but doesn't load videos.
